docs/how-to-guides/platform/custom-block-editor/tutorial.md

```diff
@@ -435,12 +433,8 @@ Jumping back to our own custom `<BlockEditor>` component, it is also worth notin
 
 <div className="editor-styles-wrapper">
 	<BlockEditorKeyboardShortcuts /> /* 1. */
-	<WritingFlow>
-		/* 2. */
-		<ObserveTyping>
-			/* 3. */
-			<BlockList className="getdavesbe-block-editor__block-list" />
-		</ObserveTyping>
+	<WritingFlow> /* 2. */
```

Comment:
Next step :P?

Reply:
Maybe! I've been thinking about this one and there's several things I'd like to change here. The focus capture element should ideally render outside the iframe as well.